app/controllers/articles_controller.rb

```diff
@@ -27,7 +28,7 @@ def create
   end
 
   def edit
-    @article = Article.find(params[:id])
+    @article
```

Comment:
@jshidalgokommit If you do` @article = Article.find(params[:id])` on the `verify_owner` function, you don't need this line